pkg/cmd/taskrun/describe.go

```diff
@@ -104,6 +109,13 @@ or
 				opts.TaskrunName = args[0]
 			}
 
+			if opts.Status {
```

Comment:
should we do error handling in case of `--steps` is mentioned

Reply:
Sure. I think it's reasonable.